ui/src/tugboat/components/ImageUpload.tsx

```diff
@@ -0,0 +1,283 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { Upload, message } from 'antd';
+import { UploadChangeParam } from 'antd/lib/upload';
+import { UploadFile } from 'antd/lib/upload/interface';
+import { LoadingOutlined, UploadOutlined } from '@ant-design/icons';
+
+export interface ModelParams {
```

Comment:
```suggestion
export interface UploadedImage {
```

Or maybe `ImageData`, or something like that.  In my opinion there's nothing model specific about this, so a more general name for the type makes sense.

Reply:
doh, i meant to do this before making the pr!

Reply:
done